coderd/database/migrations/000232_update_dormancy_notification_template.up.sql

```diff
@@ -0,0 +1,16 @@
+UPDATE notification_templates
+SET
+	body_template = E'Hi {{.UserName}}\n\n' ||
+		E'Your workspace **{{.Labels.name}}** has been marked as [**dormant**](https://coder.com/docs/templates/schedule#dormancy-threshold-enterprise) because of {{.Labels.reason}}.\n' ||
+		E'Dormant workspaces are [automatically deleted](https://coder.com/docs/templates/schedule#dormancy-auto-deletion-enterprise) after {{.Labels.timeTilDormant}} of inactivity.\n' ||
+		E'To prevent deletion, use your workspace with the link below.'
+WHERE
+	id = '0ea69165-ec14-4314-91f1-69566ac3c5a0';
+
+UPDATE notification_templates
+SET
+	body_template = E'Hi {{.UserName}}\n\n' ||
+		E'Your workspace **{{.Labels.name}}** has been marked for **deletion** after {{.Labels.timeTilDormant}} of [dormancy](https://coder.com/docs/templates/schedule#dormancy-auto-deletion-enterprise) because of {{.Labels.reason}}.\n' || 
```

Comment:
I'm not sure how this notification was tested before submitting your PR because this label was misnamed.

Reply:
I was testing it only using the tests, but I can see how flaky it is. I'm starting to wonder if we should test the notifications using the database, so we can catch any mismatches between the template body and provided labels.

This is kind of related to my comment [here](https://github.com/coder/coder/pull/14029#issuecomment-2253287566).

Reply:
Agreed, but let's not make it a condition of this PR.
I think we're aligned given what I requested in https://github.com/coder/coder/pull/14029#pullrequestreview-2204224750.